.github/workflows/tox.yml

```diff
@@ -1,5 +1,8 @@
 name: Python CI
 on: [push, pull_request]
+concurrency:
+  group: tox
```

Comment:
Don't you need to include something that identifies a reference, so you don't cancel other PRs ?
We're using `group: <something>-${{ github.ref }}` in Galaxy for that reason.

Reply:
You're obviously right, I copy/pasted but deleted too much of the line. Fixed in a rebase.